package.json

```diff
@@ -51,6 +54,6 @@
     "lint": "eslint '**/*.js'",
     "lint:fix": "npm run lint -- --fix",
     "serve": "nodemon backend/index.js",
-    "test": "mocha backend/test"
+    "test": "mocha backend/test --exit"
```

Comment:
Interval was used before, not sure why it hangs now? Maybe it's possible to cancel interval when there are no online players?